### PR TITLE
Feature/ds 383

### PR DESCRIPTION
**Pull request by rollki**

### CODE REVIEW

**Comment by anpolimus** [comment]
Hi, @rollki . Is this PR is ready for review?

**Comment by rollki** [comment]
Hi @anpolimus 
I pushed into the repo the developed functionality, and it's working but still needs some refactoring (endpoints, creating and editing events - need to change the existing code in the distribution to be able to reuse it).

File: README.md [hunk 1]
```diff
@@ -1 +1,58 @@
...
+## Installation
+
+1. Download and install the module in your Drupal project.
+2. Enable the module by navigating to `Admin > Extend` in your Drupal admin interface.
+
+## Usage
+
+### Displaying the Calendar
+
+To display the calendar, create a new page and use the provided controller:
```
Comment (anpolimus) [inline README.md:14]: @rollki maybe you mean use block?
How we can use controller on the landing page?

File: fullcalendar-app/src/components/FullCalendar.vue [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+    CalendarHeader,
+  },
+  data() {
+    return {
+      activeModal: null,
+      popoverStyle: {},
+      selectedEvent: null,
+      selectedCategories: [],
+      categories: [
+        "Academic Enrichment - Youth",
```
Comment (anpolimus) [inline fullcalendar-app/src/components/FullCalendar.vue:65]: We have to un-hard code this.

File: y_pef_schedule.module [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * @file
+ * Implements hooks for the y_pef_schedule module.
+ */
+
+declare(strict_types = 1);
```
Comment (anpolimus) [inline y_pef_schedule.module:8]: Is it something new in the PHP files?
Havent seen it is used in Drupal code.

**Review by anpolimus — COMMENTED** [review]
Hi, @rollki. In general everything is fine.
It would be great to have a call and discuss how finalise this feature.

File: y_pef_schedule.module (hunk 3 above)
Comment (rollki) [reply y_pef_schedule.module:8]: It seems it came with PHP 7 and is recommended for use everywhere.

**Comment by rollki** [comment]
Closed since a new PR from main branch was created https://github.com/YCloudYUSA/y_pef_schedule/pull/3

**Pull request closed without merging**